examples/ui.content/src/content/jcr_root/content/core-components-examples/library/form-button/.content.xml

```diff
@@ -0,0 +1,293 @@
+                    descriptionFromPage="false"
+                    fileReference="/content/dam/core-components-examples/library/github-logo.svg"
+                    linkURL="https://github.com/adobe/aem-core-wcm-components/tree/master/content/src/content/jcr_root/apps/core/wcm/components/form/button/v2/button"
+                    textIsRich="true"
+                    titleFromPage="false"/>
+                <teaser_556024830
+                    cq:styleIds="[1550165685463]"
+                    jcr:created="{Date}2019-02-14T16:41:59.595+01:00"
+                    jcr:createdBy="admin"
+                    jcr:description="&lt;p>Adobe Help Center&lt;/p>&#xd;&#xa;"
+                    jcr:lastModified="{Date}2019-12-19T16:00:24.470+01:00"
+                    jcr:lastModifiedBy="admin"
+                    jcr:primaryType="nt:unstructured"
+                    jcr:title="Using Core Components"
+                    sling:resourceType="core/wcm/components/teaser/v1/teaser"
+                    actionsEnabled="false"
+                    descriptionFromPage="false"
+                    fileReference="/content/dam/core-components-examples/library/adobe-logo.svg"
+                    linkURL="https://docs.adobe.com/content/help/en/experience-manager-core-components/using/components/forms/form-button.html"
+                    textIsRich="true"
+                    titleFromPage="false"/>
+                <title_865328580
+                    cq:styleIds="[1544759664489]"
+                    jcr:created="{Date}2018-12-06T19:22:23.729+01:00"
+                    jcr:createdBy="admin"
+                    jcr:lastModified="{Date}2018-12-10T17:08:09.903+01:00"
+                    jcr:lastModifiedBy="admin"
+                    jcr:primaryType="nt:unstructured"
+                    jcr:title="Examples"
+                    sling:resourceType="core/wcm/components/title/v2/title"
+                    type="h2"/>
+                <title_1975440126
+                    cq:styleIds="[1544759676459]"
+                    jcr:created="{Date}2019-12-12T15:09:49.918+01:00"
+                    jcr:createdBy="admin"
+                    jcr:lastModified="{Date}2019-12-19T16:07:33.964+01:00"
+                    jcr:lastModifiedBy="admin"
+                    jcr:primaryType="nt:unstructured"
+                    jcr:title="Form Button"
+                    sling:resourceType="core/wcm/components/title/v2/title"/>
+                <text_1068966881
+                    jcr:created="{Date}2019-12-12T15:10:07.890+01:00"
+                    jcr:createdBy="admin"
+                    jcr:lastModified="{Date}2019-12-19T16:55:45.391+01:00"
+                    jcr:lastModifiedBy="admin"
+                    jcr:primaryType="nt:unstructured"
+                    sling:resourceType="core/wcm/components/text/v2/text"
+                    text="&lt;p>Form Container Form Options and a Form Button as child elements.&amp;nbsp;&lt;/p>&#xd;&#xa;"
```

Comment:
@cleliameneghin - we should probably simplify this page and not wrap the Form Button by a container, as the demo code sections are incorrectly displaying the markup and properties for the container, not the button.

Reply:
sadly it seems this is not possible. When I remove the form container around the form-options & the form-button, form-button disappears. 
Only options are shown. In crxde button is still there but not shown anymore